src/pages/NewChatPage.js

```diff
@@ -107,22 +109,24 @@ class NewChatPage extends Component {
             title: this.props.translate('common.recents'),
             data: recentReportsWithoutSelected,
             shouldShow: !_.isEmpty(recentReportsWithoutSelected),
-            indexOffset: _.reduce(sections, (prev, {data}) => prev + data.length, 0),
+            indexOffset,
         });
+        indexOffset += recentReportsWithoutSelected.length;
```

Comment:
rather than having to iterate through all the previous sections to get the `indexOffset`, it's much simpler if we store the `indexOffset` in a variable and increment it as we go. It's also more legible IMO

Reply:
Clearly something with the old design was hard to understand since the indexOffset was incorrectly calculated for the last added section

Reply:
Updated it for all the pages that have a `getSections()` and that previously used `_.reduce` to calculate the `indexOffset`